bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I ran into this recently when testing against Lwt's latest master branch - I had a package installed that had an (arguably overly pessimistic) constraint of `lwt < 4.0.0` so the pin didn't work.  I had to modify the other package to remove the upper bound on the Lwt version constraint.